### PR TITLE
[BUGFIX] Restore post-processing of tables after a truncate

### PR DESCRIPTION
**Pull request by oliverklee**
This commit restores a line that accidentally was removed in #264.

### CODE REVIEW

File: Classes/Core/Testbase.php [hunk 1]
```diff
@@ -701,6 +701,7 @@ private function truncateAllTablesForMysql(): void
             if ($isChanged) {
                 $tableName = $tableData['table_name'];
                 $connection->truncate($tableName);
+                self::resetTableSequences($connection, $tableName);
```
Comment (lolli42) [inline Classes/Core/Testbase.php:704]: We actually dropped that by intention: This resetTableSequences() call is only needed for postgres, it's obsolete with mysql.

**Review by lolli42 — CHANGES_REQUESTED** [review]

**Comment by lolli42** [comment]
I'd keep it as is. Fine with you, Oliver?

File: Classes/Core/Testbase.php (hunk 1 above)
Comment (oliverklee) [reply Classes/Core/Testbase.php:704]: Ah, okay. Thanks for the explanation!

Should we make this PostgreSQL-only then (in a separate PR)?

Comment (oliverklee) [reply Classes/Core/Testbase.php:704]: Ah, it already is.

Comment (lolli42) [reply Classes/Core/Testbase.php:704]: Unsure. There's a switch for sqlite in there, too. It's just that it was clearly obsolete for mysql/maria.

**Pull request closed without merging**